test/storage/sequelize.test.ts

```diff
@@ -8,6 +8,12 @@ import { join } from 'path';
 import { v4 as uuid } from 'uuid';
 import jetpack = require('fs-jetpack');
 
+// TODO [sequelize@>=6.0.0] remove when sequelize fixes the types bug
+// a change in the types of sequelize@next makes `describe` a static function
+// eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+// @ts-ignore
+const describeModel = (model: sequelize.Model) => model.describe();
```

Comment:
I think ts-ignore is marginally better, since it doesn't lose type information with the current version of sequelize, which is what's installed by default.

Reply:
Ah, ok!